### PR TITLE
updating the two orphaned packages in JS and Java. .NET seems intenti…

### LINKED ISSUE #4438: [github.io] Standardize `Core` sections into `Other` category
This is prompted by this by PR #4411

It looks like we need to do a standardization pass on all of the main languages. End result, our `Core` github.io page should entirely disappear and be supplanted by `Other` categories that look the ones from docs.ms.com.

- [x] https://azure.github.io/azure-sdk-for-python/core.html
- [ ] https://azure.github.io/azure-sdk-for-net/core.html
- [x] https://azure.github.io/azure-sdk-for-js/core.html
- [x] https://azure.github.io/azure-sdk-for-java/core.html

Basically, we don't want to have this situation:

image

In _any_ of our primary languages. Everything should exist under `Other` categories like:

image

Properly categorizing everything starting from the `github.io` side will be the easiest way to account for this.

### PR DESCRIPTION
**Pull request by scbedd**
Resolves #4438 

@sima-zhu @danieljurek who should I talk to about .NET? They only do the the `Core` service. Nothing in `Other` afaict.

As it stands, this PR will result in the elimination of these two pages (the single item therein will move to `Other`)

- https://azure.github.io/azure-sdk-for-js/core.html
- https://azure.github.io/azure-sdk-for-java/core.html

### CODE REVIEW

**Review by deyaaeldeen — APPROVED** [review]
JS change looks good

**Comment by jsquire** [comment]
> .NET still has some core packages left in core instead of Other. Can we move them in the same PR?

I'm good with this.

**Comment by scbedd** [comment]
@jsquire I will update the PR!

**Comment by scbedd** [comment]
This is the current [core github.io](https://azure.github.io/azure-sdk-for-net/core.html) page.

image

Every package that appears in the image above has been recategorized in my newest commit.

I am have updated to follow the `core` patterns to try to recreate the same. @sima-zhu @danieljurek can you please review the new commit? I'm pretty certain I interpreted the ToC automation properly.

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -21,8 +21,8 @@
 "Azure.Security.ConfidentialLedger","","1.0.0-beta.2","ConfidentialLedger","ConfidentialLedger","confidentialledger","","","client","true","","","","","","","","",""
 "Azure.Containers.ContainerRegistry","1.0.0","1.1.0-beta.4","Container Registry","Container Registry","containerregistry","","","client","true","","01/14/2022","","","","","","",""
 "Azure.AI.Language.Conversations","","1.0.0-beta.3","Conversation Analysis","Cognitive Language Understanding","cognitivelanguage","","","client","true","","","","","","","","",""
-"Azure.Core","1.24.0","","Core","Core","core","","","client","true","","10/29/2019","active","","","","","",""
-"Azure.Core.Amqp","1.2.0","","Core - AMQP","Core","core","","","client","true","","11/23/2020","active","","","","","",""
+"Azure.Core","1.24.0","","Other - Core","Other","core","","","client","true","","10/29/2019","active","","","","","",""
```
Comment (sima-zhu) [inline _data/releases/latest/dotnet-packages.csv:24]: image
The 4th should be `Core - client - {package name}`. This is for docs.ms toc structure

Comment (sima-zhu) [reply _data/releases/latest/dotnet-packages.csv:24]: Same as below

File: _data/releases/latest/js-packages.csv [hunk 2]
```diff
@@ -8,7 +8,7 @@
 "@azure/schema-registry-avro","1.0.0","","Avro","Schema Registry","schemaregistry","","","client","true","","05/10/2022","","","","","","",""
 "@azure/eventhubs-checkpointstore-blob","1.0.1","","Azure Blob Storage Checkpoint Store","Event Hubs","eventhub","","","client","true","","01/09/2020","active","","","","","",""
 "@azure-rest/confidential-ledger","","1.0.0-beta.2","Azure Confidential Ledger","Azure Confidential Ledger","https://github.com/Azure/azure-sdk-for-js/tree/@azure-rest/confidential-ledger_item.Version/sdk/confidentialledger/confidential-ledger-rest","NA","NA","client","true","","","","","","","","",""
-"@azure/core-http-compat","1.3.0","","Azure Core Http Compat","Core","core","","","client","true","","03/29/2022","","","","","","",""
+"@azure/core-http-compat","1.3.0","","Azure Core Http Compat","Other","core","","","client","true","","03/29/2022","","","","","","",""
```
Comment (danieljurek) [inline _data/releases/latest/js-packages.csv:11]: Depending on where you want this nested in the ToC you may need to update the `DisplayName` field to match the nesting you want: 

For example, `@azure/abort-controller` lives under `Other/Core/Client` 
image

image


It has the `DisplayName` value of `
Core - Client - Abort Controller`

**Review by danieljurek — APPROVED** [review]
I think the JS package `DisplayName` might need to be something like `Core - Client - Http Compat` 

Other adjustments LGTM. We haven't done a thorough cleanup of .NET metadata yet but this is a step in the right direction.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (scbedd) [reply _data/releases/latest/dotnet-packages.csv:24]: I don't understand what you're telling me to do here.

Comment (scbedd) [reply _data/releases/latest/dotnet-packages.csv:24]: I don't _want_ this to be under `Core`

File: _data/releases/latest/js-packages.csv (hunk 2 above)
Comment (scbedd) [reply _data/releases/latest/js-packages.csv:11]: ok. So the displayname only impacts below the targeted Service. Which is the 5th column (index 4).

Comment (scbedd) [reply _data/releases/latest/js-packages.csv:11]: I was definitely not understanding that. I added `Other - <blah` in front of all the .NET updates.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (scbedd) [reply _data/releases/latest/dotnet-packages.csv:24]: Ok I think I get it. The Displayname is what is _below_ the value of column 5. 

So `Other` will be the top level on the ToC.

Then, we split by `-` and nest.

`Core - Client - Http Netty`

on actual ToC will be

```
Other
   Core
      Client
         Http Netty
            Overview
            <auto-generated>
```

Is my understanding on this correct?

File: _data/releases/latest/js-packages.csv (hunk 2 above)
Comment (scbedd) [reply _data/releases/latest/js-packages.csv:11]: I'll walk the update with that new understanding. Thanks.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (sima-zhu) [reply _data/releases/latest/dotnet-packages.csv:24]: That is correct

**Review by danieljurek — APPROVED** [review]

**Pull request merged**